src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml

```diff
@@ -71,7 +65,17 @@
 
             <!--  Show Titlebar  -->
             <local:SettingContainer x:Uid="Globals_ShowTitlebar">
-                <ToggleSwitch IsOn="{x:Bind ViewModel.ShowTabsInTitlebar, Mode=TwoWay}"
+                <ToggleSwitch x:Name="ShowTitlebarToggleSwitch"
+                              IsOn="{x:Bind ViewModel.ShowTabsInTitlebar, Mode=TwoWay}"
+                              Style="{StaticResource ToggleSwitchInExpanderStyle}"
+                              Toggled="{x:Bind local:GlobalAppearanceViewModel.ShowTitlebarToggled}" />

```

Comment:
I think that it will work if you use `{x:Bind ViewModel.ShowTitlebarToggled}`! This way, you are using the XAML local variable `ViewModel` instead of the global variable `local:GlobalAppearanceViewModel`.

I know that calling the `local:` one a "global" variable is confusing. It is because it is declared on line 9 in this file as being an _XML namespace_ for this whole file.